src/bindings/python/pyproject.toml

```diff
@@ -1,3 +1,40 @@
+[project]
+name = "openvino"
+dynamic = ["version", "entry-points", "scripts"]
+description = "OpenVINO(TM) Runtime"
+readme = {file = "README.md", content-type="text/markdown"}
+license = {text = "OSI Approved :: Apache Software License"}
+authors = [
+    { name = "OpenVINO Developers", email = "openvino@intel.com" },
```

Comment:
does openvino@intel.com exist?

Reply:
It should, we provide it now in our wheels)